features/external_authorization.md

```diff
@@ -154,49 +154,51 @@ N/A. Design doc presented instead.
 
 **Design**
 
-- [ ] Design doc describing the intention of the feature, how it will be
+- [x] Design doc describing the intention of the feature, how it will be
 	implemented, and any thoughts on how to test the feature has been approved by
 	relevant work group leads
-- [ ] Feature coverage and test plans written and approved.
+> Design doc: https://docs.google.com/document/d/1V4mCQCw7mlGp0zSQQXYoBdbKMDnkPOjeyUb85U07iSI/edit
+- [x] Feature coverage and test plans written and approved.
 
 **Docs** 
 
-- [ ] Documentation on istio.io includes performance expectations; may have caveats. 
-- [ ] Documentation on istio.io includes samples/tutorials. 
-- [ ] Documentation on istio.io includes appropriate glossary entries. 
-- [ ] All new documentation containing user actions includes istio.io tests.
-- [ ] Release notes have been added. 
-- [ ] Upgrade notes have been added. 
+- [x] Documentation on istio.io includes performance expectations; may have caveats. 
```

Comment:
I don't see any performance expectations listed in https://istio.io/latest/docs/tasks/security/authorization/authz-custom/ , should I be looking elsewhere?

Reply:
Performance tests are here: https://github.com/istio/tools/tree/master/perf/benchmark/configs/istio/ext-authz

Reply:
@therealmitchconnors Any other feedback?

Reply:
I spoke with @bochengchu offline, and it looks like we need to push a change to docs to include performance expectations.

Reply:
@therealmitchconnors Does https://github.com/istio/istio.io/pull/12176 satisfy this?

Reply:
I think so, though we may want to consider whether we should have istio.io docs pointing to GitHub.  But one way or another that content should be available, which seems to satisfy this.